include/mitsuba/python/docstr.h

```diff
@@ -754,6 +754,13 @@ inexpensively. When this is not possible, the value is approximated by
 evaluating the BSDF for a normal outgoing direction and returning this
 value multiplied by pi. This is the default behaviour of this method.
 
+Parameter ``si``:
+    A surface interaction data structure describing the underlying
+    surface position.)doc";
+
+static const char *__doc_mitsuba_BSDF_eval_roughness =
+R"doc(Evaluate the roughness
+
```

Comment:
Can you rerun the `docstrings` target please?  This is currently missing some of the documentation you wrote :/

Reply:
Sorry I am not sure I understand what you mean by rerunning `docstrings`?